File: multi_lang/java_templates/test_template.java
// Template: JUnit 5 tests (raw). Placeholders substituted by JavaTestGenerator.
package __PACKAGE_NAME__;

import static org.junit.jupiter.api.Assertions.assertEquals;
import org.junit.jupiter.api.Test;

class __CLASS_NAME__ControllerTest {

  @Test
  void healthzReturnsOk() {
    String body = "{\"status\":\"ok\"}";
    assertEquals("ok", "ok");
    assertEquals(true, body.contains("ok"));
  }
}

File: multi_lang/rust_templates/test_template.rs
// Template: cargo test unit tests (raw). Placeholders substituted by RustTestGenerator.
#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn healthz_returns_ok() {
        let body = "{\"status\":\"ok\"}";
        assert!(body.contains("ok"));
    }

    #[test]
    fn list__MODEL_NAME___responds() {
        let route = "/api/v1/__ROUTE__";
        assert_eq!(route, "/api/v1/__ROUTE__");
    }
}